packages/rs-dpp/src/fee/fee_result/mod.rs

```diff
@@ -192,6 +193,19 @@ impl FeeResult {
             removed_bytes_from_system: 0,
         }
     }
+
+    /// Apply a fee multiplier to a fee result
+    pub fn apply_fee_multiplier(&mut self, fee_multiplier: FeeMultiplier) {
+        let additional_processing_fee = (self.processing_fee as u128)
+            .saturating_mul(fee_multiplier as u128)
+            .saturating_div(100);
+        if additional_processing_fee > u64::MAX as u128 {
+            self.processing_fee = u64::MAX;
+        } else {
+            self.processing_fee += additional_processing_fee as u64;
```

Comment:
couldn't this overflow self.processing here? above should be something like `if self.processing_fee  + additional_processing_fee > u64::MAX as u128` no?

Reply:
Good catch.